sites/fast-website/src/app/components/card-section/card-section.template.ts

```diff
@@ -0,0 +1,4 @@
+import { html } from "@microsoft/fast-element";
+import { CardSection } from "./card-section";
+
+export const CardSectionTemplate = html<CardSection>` <slot></slot> `;
```

Comment:
are the spaces intended before and after slots?

Reply:
This is a good catch, I think this is going to actually include spaces within the template, which (I think) is unintended.

Reply:
Good question.  I had that on separate lines so prettier must have moved it to one line.  I'll remove the spaces.